src/main/java/uk/gov/companieshouse/overseasentitiesapi/utils/BeneficialOwnerTypes.java

```diff
@@ -0,0 +1,29 @@
+package uk.gov.companieshouse.overseasentitiesapi.utils;
+
+import uk.gov.companieshouse.overseasentitiesapi.model.dao.trust.BeneficialOwnerType;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+
+public class BeneficialOwnerTypes {
+
+    private BeneficialOwnerTypes() {
+    }
+
+    public static final Map<String, BeneficialOwnerType> beneficialOwnerTypes = initMap();
+
+    public static Map<String, BeneficialOwnerType> initMap() {
+        Map<String, BeneficialOwnerType> map = new HashMap<>();
+        map.put("Corporate Interested Person", BeneficialOwnerType.CORPORATE_INTERESTED_PERSON);
```

Comment:
This is confusing me a bit. Wondering what the HashMap is used for but it also seems that the string key values match the enum string values, so could you reference the enum for them too? Which then begs the question, what is this used for?